tools/devops/automation/templates/tests/stage.yml

```diff
@@ -84,8 +84,6 @@ stages:
 - stage: ${{ parameters.stageName }}
   displayName: ${{ parameters.displayName }}
   dependsOn:
-  - ${{ if eq(parameters.postPipeline, false) }}:
-    - build_macos_tests
```

Comment:
I believe they must wait for the normal build to finish though, otherwise none of the packages will be built.

Reply:
True, so there is a something fishy there, it should not be build_macos_tests, but build the packages